### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Connect to Local Server_Keyboard: Focus order is not logical on the page.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#98]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**MAS ID Violated:** MAS2.4.3

**Repro Steps:**
1. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2. Download and install "Microsoft IIS Administration" application.
3. Tab till "Skip This" button and Press Enter.
4. Verify that focus order is proper and logical throughout the page.

**Actual Result:**
Keyboard focus is coming to "Get Microsoft IIS Administration" at last after navigating to all controls of the page, which is not logical.

**Expected Result:**
While navigating through keyboard, focus should move from "not connected" link to "Get Microsoft IIS Administration" link and then to "Access Token" input field. 

**User Impact:**
"Get Microsoft IIS Administration" link should get the focus first, because without downloading the IIS Administration software, user will be unable to perform any other task on this page.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

**Attachment:**
[Bug 98_focus order is not logical on the page.pptx](https://github.com/Microsoft/IIS.WebManager/files/2323548/Bug.98_focus.order.is.not.logical.on.the.page.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Accessibility56** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #C+E-H; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #MAS2.4.3; #Keyboard; #.NETWebsites; #Edge; #Chrome

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
